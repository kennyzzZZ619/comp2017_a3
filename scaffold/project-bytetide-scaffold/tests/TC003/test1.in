ident:0000000000111111111122222222223333333333444444444455555555556666666666777777777788888888889999999999000000000011111111112222222222333333333344444444445555555555666666666677777777778888888888999999999900000000001111111111222222222233333333334444444444555555555566666666667777777777888888888899999999990000000000111111111122222222223333333333444444444455555555556666666666777777777788888888999999999990000000000111111111122222222223
filename:file1.bpkg
size:103483
nhashes:6
hashes:
	f90c0f7c80fc66034a29cd1c50f279e31865ce2d67fae918a207319d58d10752
	96e76decae159031853de16c92b91d9e8a86de625baa79527c39f0b64d1c18e4
	b29f5422b580f7e5c91474fe34cd74b59a0308cd26989a8b4a630b9c4f3d64a1
	b177eaa8156391eb1834022666e571c09a527463d65ffa11c5b758e6fe72d9f9
	34d555731274f2179cad42ad3c2f991901964365ae766aaba0cae88b6fbd6837
	52fe1187d34c74ccb657f211562c01ce8b927bacc952a93dafba3a56d0ed29f2
nchunks:12
chunks:
	3192eaec89b7ce3a26c5ddbe779e373aacbd7d884e4b1019000270a5284b38bb,987136,4096
	6517828b6ee21d7436c549012ced9be574c74bbeb2cb5634f699890160ebcd79,991232,4096
	fd1137480c975a8515a07c2987ba68a8fc38d295c553d01609789320837290d6,995328,4096
	2274a2c8c3778f81ea90a71be3f8ba17044e1f2d2c95a46e60db7714556f3e13,999424,4096
	73c44f4b40bdf9fe136802c582e7178683c76aff723c9bda297a8fab7a432ecf,1003520,4096
	cd4703821af87013ac59d623e7efb5c7097d27fd6274665cd587f27392d88aeb,1007616,4096
	1b63b5df750806e6c2a04b46b1d0afce1eb13d1a64784141aa84e73160c76f29,1011712,4096
	2a895c1b09afcf46197b9f23a229cd2f93e1e7553497a37d7637e8986fc09e78,1015808,4096
	fba201156fc7f36b66175ebfbbe49bceee66798858a1bf6b7ff3ded91d211463,1019904,4096
	b9b3e9af1f6f3867cf5baa55db1a9047cbc28efa4c3af376f01a7e287989cdb8,1024000,4096
	a109cad909a42cab544b8f5729ea1b81f84e50b4f19cad8be41c6e11395128bf,1028096,4096
	c86a68d7ea86a6041a506e91eb9debae107ba93e4e1e84d6c1953992970b670e,1032192,4096
