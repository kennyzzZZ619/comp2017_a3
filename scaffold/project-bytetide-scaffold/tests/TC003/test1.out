Error: Identifier exceeds the maximum length of 1024 characters.
